search/services/index/__init__.py

```diff
@@ -11,6 +11,24 @@
 :mod:`search.agent.consumer.MetadataRecordProcessor`).
 """
 
+
+# Start monkeypatch of elasticsearch-py's search(), use POST not GET
+
```

Comment:
Looks great.

Can you add something like this to the comment: "due to GCP load balancers rejecting any GET requests with a body"